presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -2,12 +2,11 @@
 LDAP Authentication
 ===================
 
-Presto can be configured to enable frontend LDAP authentication over
-HTTPS for clients, such as the :ref:`cli_ldap`, or the JDBC and ODBC
-drivers. At present, only simple LDAP authentication mechanism involving
-username and password is supported. The Presto client sends a username
-and password to the coordinator, and the coordinator validates these
-credentials using an external LDAP service.
+Presto can be configured to enable frontend LDAP authentication for clients,
+such as the :ref:`cli_ldap`, or the JDBC and ODBC drivers. At present, only
```

Comment:
Remove ODBC driver since there is no open source ODBC driver. Make the JDBC driver mention a link.

Reply:
Let’s leave this alone for now. Changing the text here for ODBC is out of scope for unsecured LDAP.